doc/classes/Node.xml

```diff
@@ -387,6 +387,13 @@
 				Returns [code]true[/code] if the node is folded (collapsed) in the Scene dock.
 			</description>
 		</method>
+		<method name="is_editable_instance" qualifiers="const">
+			<return type="bool" />
+			<argument index="0" name="node" type="Node" />
+			<description>
+				Returns [code]true[/code] if [code]node[/code] has editable children enabled relative to this node.
```

Comment:
I would add in both documentation items that this is meant to be useful only for editor tooling, so they understand the methods do nothing outside the editor.